lighthouse-core/report/v2/renderer/performance-category-renderer.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* globals self, Util */
+
+class PerformanceCategoryRenderer extends CategoryRenderer {
```

Comment:
@paulirish `category-renderer.js` was huuuuge. I started refactor by splitting it up into three files (perf, a11y, default). Hope that's OK? In the next step I'll move audit group support from a11y to the default renderer and (hopefully?) nuke the a11y render.

Reply:
sounds great